src/module.js

```diff
@@ -190,7 +201,19 @@ const PARTICIPANT_PROPS = {
     help: 'styles format should be a subset of: ' +
           '{ cam: {div: {}, video: {}}, screen: {div: {}, video: {}} }'
   },
-  setAudio: true, setVideo: true, eject: true
+  setSubscribedTracks: {
+    validate: (subs) => {
+      for (const s in subs) {
+        if (!(s === 'audio' || s ===  'video' || s === 'screenVideo')) {
```

Comment:
I think it would be really nice to support the option for `all` on the per-participant basis. It looks like it would be easy enough to support and then a user can think mostly about simply subscribing or unsubscribing to/from a participant vs which participants and which streams.

Reply:
Great idea. Will implement. I put it in some draft docs notes here: https://www.notion.so/dailyco/d56d6561a3d949d6ad0c14464292c2b2?v=9546498c730e4e6b9f20ee593d9ce60d&p=9c32bec8274f4f239cb73f92c335df92

Reply:
done